Neos.Flow/Classes/Mvc/Routing/Router.php

```diff
@@ -128,7 +131,7 @@ public function route(RouteContext $routeContext): array
                 return $matchResults;
             }
         }
-        $this->logger->notice(sprintf('Router route(): No route matched the route path "%s".', $httpRequest->getRelativePath()));
+        $this->logger->debug(sprintf('Router route(): No route matched the route path "%s".', $httpRequest->getRelativePath()));
```

Comment:
Maybe info is enought, you can filter out info level on production, debug seems a bit like "we really don't care about this in production"

Reply:
Frankly, I don't care about that in production. It just says, the url could not be resolved, which leads to a 404. I care about the 404s in the web servers access log, but I don't need it in the system log of the application. 

And maybe I do it wrong, but I do care about meaningful info log messages in production. They tell me what is happening / what happened on the system which not necessarily needs to be an error.